docs/zh_CN/debugging/README.md

```diff
@@ -0,0 +1,59 @@
+# MinIO Server 调试教程 [![Slack](https://slack.min.io/slack?type=svg)](https://slack.min.io) [![Docker Pulls](https://img.shields.io/docker/pulls/minio/minio.svg?maxAge=604800)](https://hub.docker.com/r/minio/minio/)
+
+### HTTP Trace
+可以通过 [`mc admin trace`](https://github.com/minio/mc/blob/master/docs/minio-admin-complete-guide.md#command-trace---display-minio-server-http-trace) 命令启用HTTP跟踪。
+
+示例:
+```sh
+minio server /data
+```
+
+默认只跟踪API调用操作和HTTP响应状态。
+```sh
+mc admin trace myminio
+```
+
+跟踪整个HTTP请求 
+```sh
+mc admin trace --verbose myminio
+```
+
+跟踪整个HTTP请求和节点间通信
+```sh
+mc admin trace --all --verbose myminio
+```
+
+
+### 诊断系统
```

Comment:
Excuse my ignorance as I only speak english, according to google translate 诊断系统 (diagnosis) is different than 诊断程序 (diagnostics).  Calling it out in case there is a distinct difference that needs to be changed

Reply:
The difference is not big, it is understandable, strictly speaking, the diagnostic tool is more suitable, because odb belongs to minio . @eco-minio